src/sinol_make/commands/run/__init__.py

```diff
@@ -706,6 +706,42 @@ def validate_arguments(self, args):
 
         return compilers, timetool_path
 
+    def set_scores(self):
+        self.tests = self.get_tests(self.args.tests)
+        self.groups = list(sorted(set([self.get_group(test) for test in self.tests])))
+        self.scores = collections.defaultdict(int)
+
+        if 'scores' not in self.config.keys():
+            print(util.warning('Scores are not defined in config.yml. Points will be assigned equally to all groups.'))
+            num_groups = len(self.groups)
+            self.scores = {}
+            if self.groups[0] == 0:
+                num_groups -= 1
+                self.scores[0] = 0
+
+            points_per_group = 100 // num_groups
+            for group in self.groups:
+                if group == 0:
+                    continue
+                self.scores[group] = points_per_group
+
+            if points_per_group * num_groups != 100:
+                self.scores[self.groups[-1]] += 100 - points_per_group * num_groups
+        else:
+            for group in self.config["scores"]:
+                self.scores[group] = self.config["scores"][group]
+
+        self.possible_score = self.get_possible_score(self.groups)
+
+        print("Scores:")
```

Comment:
Does the scores get printed each time `sinol-make run` is run? I would suggest to avoid printing too much data, especially those that are easily accessible (e.g. by reading `config.yml`) and not produced by `sinol-make`.

Reply:
So maybe the program should print it only when `scores` is not defined in config?

Reply:
A suitable warning could be enough (one that e.g. explains what has changed and where to look for the changes). But if you think it would be better to print the whole `scores`, go for it.